frontend/src/components/explorer/gemBrowser/GemSearch.vue

```diff
@@ -26,7 +26,6 @@
           data-hj-whitelist
           type="text"
           class="input"
-          :placeholder="placeholder"
```

Comment:
I'm a bit confused. Have we decided to remove the placeholder completely or just remove the formula?

Reply:
I removed it completely, but I was not present during the discussion today so please add a commit as you see fit if you don't think this is what was suggested